{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:socket:engine.io:*:*:*:*:*:node.js:*:*"
        ],
        "fix_commit": "https://github.com/socketio/engine.io/commit/425e833ab13373edf1dd5a0706f07100db14e3c6",
        "version_end_excluding": {
            "socket:engine.io": "3.6.1"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "TYPESCRIPT",
    "licences": {
        "database": "",
        "affected_repository": "Unknown"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2022-41940"
        ],
        "CWE": [
            "CWE-248"
        ]
    }
}